### PR TITLE
Malaria Disease Detection

### PR DESCRIPTION
**Pull request by Soumayan-pal01**
## Malaria Disease Detection

- The aim of this project is to recognize whether the image of the human cell is infected with Malaria disease or not.

- [x] LGM-SOC'21 Participant 
- [ ] Contributor

Closes: #244 

### Describe the add-ons or changes you've made

In this project we have used Convolutional neural Network approach to train a model on the provided images of uninfected and parasitized human cell. This model gives a validation accuracy of 94.89% and also detects all the images of the evaluation set with correct labels.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by prathimacode-hub — APPROVED** [review]
That was amazing representations of visualizing the cell images to classify malaria detection. You had done a splendid job. Good to see your perfection in every issue you take up. Can't agree more. 🧨🎊🤩

**Comment by Soumayan-pal01** [comment]
Thank you @prathimacode-hub 😇

**Comment by Soumayan-pal01** [comment]
Can I get to know for which level this merge will be counted in the LGM-Soc @prathimacode-hub as the level tag has not been assigned.

**Comment by prathimacode-hub** [comment]
If level tag is not assigned, it wil be done soon. Your PR is merged. So don't worry. Labels would also be assigned. @Soumayan-pal01

**Comment by Soumayan-pal01** [comment]
Okhay, Thank you for the information @prathimacode-hub . Actually was confused with the "To Do' label as I thought its assigned to me for making more updates. Anyways thanks for clearing it out 😄.

**Comment by prathimacode-hub** [comment]
> Okhay, Thank you for the information @prathimacode-hub . Actually was confused with the "To Do' label as I thought its assigned to me for making more updates. Anyways thanks for clearing it out 😄.

Most welcome. This label is for us, to keep track of the scores. That's Ok. You are free to ask any questions. @Soumayan-pal01

**Comment by Soumayan-pal01** [comment]
Yeah now I have understood. Thank you again for the clarification @prathimacode-hub .
